src/notification.rs

```diff
@@ -133,7 +148,7 @@ impl fmt::Debug for Notification {
                self.get_event(),
                self.get_button_id(),
                self.get_scene_id(),
-               self.get_notification(),
+               self.get_notification_code(),
```

Comment:
I don't mind having it showing as None most of the time; IMO this is what the "debug" output is for :) I prefer to show raw data from the notification (even if for most Notification objects it's empty) than calculated or remembered data.

alternatively we could just not show it if it's None, but this is more work.